server/database/db.js

```diff
@@ -1,18 +1,36 @@
 const mysql = require('mysql2');
+const Sequelize = require('sequelize');
 
 const con = mysql.createConnection({
   host: 'localhost',
   user: 'root',
   password: 'mikamaus'
-})
+});
 
 con.connect(function(err) {
   if (err) {
     console.error('error connecting: ' + err.stack);
     return;
   }
   console.log('connected to mysql');
-  con.query('CREATE DATABASE IF NOT EXISTS `etsymary`')
-})
+  con.query('CREATE DATABASE IF NOT EXISTS etsymary');
+  con.query('USE etsymary');
+});
 
-module.exports = con;
\ No newline at end of file
+// const connection = new Sequelize('etsymary', 'root', 'mikamaus', {
+//   host: 'localhost',
+//   dialect: 'mysql',
+//   define: {
+//     timestamps: false
+//   }
+// });
+// connection
+//   .authenticate()
+//   .then(function(err) {
+//     console.log('Connection has been established successfully.');
+//   }, function (err) {
+//     console.log('Unable to connect to the database:', err);
+//   });
+
+module.exports = con;
+// module.exports = connection;
```

Comment:
Probably worth cleaning these out before you merge so it's not laying around later.